docs/source/tutorial.rst

```diff
@@ -190,6 +190,11 @@ You can also train other type models like GGNN, SchNet or WeaveNet, and other ta
 See output of ``python train_qm9.py --help``.
 
 
+Using your own dataset
+========================
+You can use your own dataset in Chainer Chemistry.
+`example/own_dataset <https://github.com/pfnet-research/chainer-chemistry/tree/master/examples/own_dataset/>`_ shows an example code.
```

Comment:
Code is uncountable (cf. [here](https://en.oxforddictionaries.com/definition/code)). So, "example code" or "an example" would be better.